src/components/NftRewards/NftTierCard.tsx

```diff
@@ -55,11 +55,13 @@ export function NftTierCard({
     setPreviewVisible(true)
   }
 
-  const remainingSupplyText =
-    rewardTier?.remainingSupply !== undefined &&
-    rewardTier.remainingSupply < MAX_REMAINING_SUPPLY
-      ? t`${rewardTier?.remainingSupply} remaining`
-      : t`Unlimited`
+  const remainingSupply = 0 //rewardTier?.remainingSupply
+  const canSelect = remainingSupply && remainingSupply > 0
```

Comment:
**suggestion** might be clearer to call this variable `isSoldOut` wdyt?

Reply:
Yep good call